api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -191,68 +191,50 @@ public IActionResult Move(string path, string destination, [FromRoute] string? l
     [SwaggerOperation(Tags = new[] { "Storage" })]
     public IActionResult Delete(string path, [FromRoute] string? location = "capture")
     {
-        var safePath = System.IO.Path.Combine(_options.GetRootPath(location), path.MakeRelativePath());
+        var safePath = Path.Combine(_options.GetRootPath(location), path.MakeRelativePath());
         if (!safePath.FileExists() && !safePath.DirectoryExists()) throw new InvalidOperationException($"File/folder does not exist: '{path}'");
 
-        var info = new ItemModel(safePath);
+        var item = new ItemModel(safePath);
         System.IO.File.Delete(safePath);
-        return new JsonResult(info);
+        return new JsonResult(item);
     }
 
     /// <summary>
     /// Make a clip from the target file identified in the clip parameter 'prefix'.
     /// </summary>
-    /// <param name="fileName"></param>
-    /// <param name="directory"></param>
+    /// <param name="path"></param>
     /// <param name="start"></param>
     /// <param name="end"></param>
-    /// <param name="clipNbr"></param>
-    /// <param name="prefix"></param>
+    /// <param name="outputName"></param>
     /// <returns></returns>
-    [HttpGet("clip")]
+    [HttpPost("clip")]
     [Produces("application/json")]
-    [ProducesResponseType(typeof(FolderModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ItemModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType((int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Storage" })]
-    public IActionResult CreateClip(string fileName, string directory, int start, int end, int clipNbr, string prefix)
+    public async Task<IActionResult> CreateClipAsync(string path, int start, int end, string outputName)
     {
-        var path = _options.CapturePath + directory;
-        var process = FfmpegHelper.GetClipProcess(fileName, path, start, end, clipNbr, prefix);
-
-        process.Start();
-        process.WaitForExit();
-
-        var listing = new FolderModel(_options.CapturePath, directory);
-        return new JsonResult(listing);
+        var safePath = Path.Combine(_options.CapturePath, path.MakeRelativePath());
+        var file = await FfmpegHelper.CreateClipAsync(safePath, start, end, outputName);
+        return new JsonResult(new ItemModel(file));
     }
 
     /// <summary>
     /// Make a clip from the target file identified in the clip parameter 'prefix'.
     /// </summary>
-    /// <param name="fileName"></param>
-    /// <param name="directory"></param>
+    /// <param name="path"></param>
     /// <param name="prefix"></param>
     /// <returns></returns>
-    [HttpPut("join")]
+    [HttpPost("join")]
```

Comment:
HTTP GET shouldn't ever be used for introducing any type of 'change'.  We're creating a new file, it should be a POST.